Sources/_OpenAPIGeneratorCore/Translator/CommonTypes/StructBlueprint.swift

```diff
@@ -80,6 +80,21 @@ struct StructBlueprint {
     var properties: [PropertyBlueprint]
 }
 
+extension StructBlueprint {
+
+    /// A Boolean value indicating whether the struct can be initialized using
+    /// an empty initializer.
+    ///
+    /// For example, when all the properties of the struct have a default value,
+    /// the struct can be initialized using `Foo()`. This is important for
+    /// other types referencing this type.
+    var hasEmptyInit: Bool {
+        // If at least one property requires an explicit value, this struct
+        // cannot have an empty initializer.
+        !properties.contains(where: { $0.defaultValue == nil })
```

Comment:
Nit: Thin it would read nicer to invert this check:

```suggestion
        properties.allSatisfy { $0.defaultValue != nil }
```

Reply:
Agreed, my fault though as I suggested the original syntax 😄

Reply:
Thank you. 🙏